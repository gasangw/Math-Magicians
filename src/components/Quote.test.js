import renderer from 'react-test-renderer';
import Quote from './quote';

it('Quote renders correctly', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});