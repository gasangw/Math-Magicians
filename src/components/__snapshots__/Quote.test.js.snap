// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div>
  <h3
    className="welcome"
  >
    Welcome to our page!
  </h3>
  <p
    className="para"
  >
    Mathematics is widely used in science for modeling phenomena. This enables the extraction of quantitative predictions from experimental laws. For example, the movement of planets can be accurately predicted using Newton's law of gravitation combined with mathematical computation.
    <br />
    <br />
    Mathematics is widely used in science for modeling phenomena. This enables the extraction of quantitative predictions from experimental laws. For example, the movement of planets can be accurately predicted using Newton's law of gravitation combined with mathematical computation
  </p>
</div>
`;

exports[`Quote renders correctly 1`] = `
<div>
  <p
    className="quote"
  >
    Mathematics is not about numbers, equations, computations,
    <br />
     or algorithms: it is about. -William Paul Thurston
  </p>
</div>
`;
